src/transformers/models/big_bird/modeling_big_bird.py

```diff
@@ -1288,9 +1319,6 @@ def __init__(self, config):
             [BigBirdLayer(config, seed=layer_idx) for layer_idx in range(config.num_hidden_layers)]
         )
 
-        # extra compared to orig-bert
-        self.LayerNorm = nn.LayerNorm(config.hidden_size, eps=config.layer_norm_eps)
```

Comment:
@vasudevgupta7, 

To comply with Bert I added this LayerNorm in the Embedding class instead. This way the models are 1-to-1 identical in terms of weight structure. You will probably need to re-convert your models so that the weights are correctly loaded. You can just run:

```bash
../transformers/src/transformers/models/big_bird/convert_bigbird_original_tf_checkpoint_to_pytorch.py --tf_checkpoint_path ./tf_checkpoint/ckpt/model.ckpt-0 --big_bird_config_file ./tf_checkpoint/config.json --pytorch_dump_path ./hf_ckpt
```